rotkehlchen/exchanges/bitfinex.py

```diff
@@ -552,34 +538,26 @@ def _deserialize_trade(
         """
         amount = deserialize_asset_amount(raw_result[4])
         trade_type = TradeType.BUY if amount >= ZERO else TradeType.SELL
-        bfx_pair = raw_result[1].replace(':', '')
-        if bfx_pair.startswith('t'):
-            bfx_pair = bfx_pair[1:]
-        bfx_fee_currency_symbol = raw_result[10]
-        if bfx_pair.startswith(bfx_fee_currency_symbol):
-            bfx_base_asset_symbol = bfx_fee_currency_symbol
-            bfx_quote_asset_symbol = bfx_pair[len(bfx_fee_currency_symbol):]
-        elif bfx_pair.endswith(bfx_fee_currency_symbol):
-            bfx_base_asset_symbol = bfx_pair[:-len(bfx_fee_currency_symbol)]
-            bfx_quote_asset_symbol = bfx_fee_currency_symbol
-        else:
+        bfx_pair = self._process_bfx_pair(raw_result[1])
+        if bfx_pair not in self.pair_bfx_symbols_map:
             raise DeserializationError(
-                f'Could not deserialize bitfinex trade pair {raw_result[1]} '
-                f'using fee_currency symbol {bfx_fee_currency_symbol}. '
+                f'Could not deserialize bitfinex trade pair {raw_result[1]}. '
                 f'Raw trade: {raw_result}',
             )
+
+        bfx_base_asset_symbol, bfx_quote_asset_symbol = self.pair_bfx_symbols_map[bfx_pair]
```

Comment:
All these changes allow getting the base and quote assets of the pair in this statement.